UPGRADE.md

```diff
@@ -10,12 +10,12 @@ Unless indicated otherwise in one of the upgrade paths of this document, it is
 possible to upgrade Kong **without downtime**:
 
 Assuming that Kong is already running on your system, acquire the latest
-version from any of the available [installation
+the version from any of the available [installation
 methods](https://getkong.org/install/) and proceed to install it, overriding
 your previous installation.
 
 If you are planning to make modifications to your configuration, this is a
-good time to do so.
+A good time to do so.
```

Comment:
incorrect change, this is a continuation of the previous line "this is a good time to do so"